tests/api/apis/supervisor/test_supervisor_routes.py

```diff
@@ -87,6 +91,7 @@ class TestSupervisorRoutes:
                 1,
                 True,
                 None,
+                MissionReaderError,
```

Comment:
Could this parameter be called something representing what it is? Like "exception_or_none" or something? "expected_exception"?